Source/Core/InputCommon/InputProfile.cpp

```diff
@@ -32,7 +32,7 @@ std::vector<std::string> GetProfilesFromSetting(const std::string& setting, cons
   std::vector<std::string> result;
   for (const std::string& setting_choice : setting_choices)
   {
-    const std::string path = root + StripSpaces(setting_choice);
+    const std::string path = root + std::string(StripSpaces(setting_choice));
```

Comment:
Is `std::string + std::string_view` invalid?

Reply:
Yes, and I can't use lioncash's `append` suggestion either since `root` is a const reference, so I think I'll just leave this one the way it is.

Reply:
Fair enough.